### PR TITLE
Standardize ruby verify building blocks

### PR DESCRIPTION
**Pull request by marklewin**
General tidy-up of Ruby building blocks as part of the Verify API doc refresh effort:

- Made `RECIPIENT_ID` and `CODE` command-line args: requires less config and also easier to use.

### CODE REVIEW

File: verify/request.rb [hunk 1]
```diff
@@ -13,15 +13,12 @@
 )
 
 response = client.verify.request(
-  number: RECIPIENT_NUMBER,
-  brand: 'Quickstart'
+  number: NEXMO_TO_NUMBER,
+  brand: 'Acme Inc'
```
Comment (mheap) [inline verify/request.rb]: Alpha senders shouldn't have a space in them

Comment (marklewin) [reply verify/request.rb]: This works in the client lib, but not in `curl`. I can only imagine that the client lib is url encoding the spaces. Let me know if you still want me to change it.

Comment (mheap) [reply verify/request.rb]: Yes please. 

> Characters outside this range (e.g. <space>) may or may not be supported depending on the destination operator.

via https://help.nexmo.com/hc/en-us/articles/217571017

**Comment by marklewin** [comment]
Done!

**Pull request merged**